gem/Rakefile

```diff
@@ -0,0 +1,30 @@
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
+
+RSpec::Core::RakeTask.new(:spec)
+
+task :default => :spec
+
+namespace :wrapper do
+  desc 'Wrap styles assets into this gem'
+  task wrap: :clean do
+    #  gs-board
+    cp_r '../dist/gs-board.html', 'app/assets/htmls'
+    cp_r '../dist/components', 'app/assets/htmls'
```

Comment:
This isn't needed since `vulcanize` creates the `gs-board.html` file with all the components bundled